modin/experimental/xgboost/test/test_dmatrix.py

```diff
@@ -74,67 +76,49 @@ def test_dmatrix_feature_names_and_feature_types(data, feature_names, feature_ty
     check_dmatrix(data, feature_names=feature_names, feature_types=feature_types)
 
 
-@pytest.mark.parametrize(
-    "feature_names",
-    [
-        ["Feature1", "Feature2", "Feature3", "Feature4", "Feature5"],
-        [u"??1", u"??2", u"??3", u"??4", u"??5"],
-    ],
-)
-def test_feature_names(feature_names):
-    data = np.random.randn(100, 5)
-    label = np.array([0, 1] * 50)
+def test_feature_names():
```

Comment:
@mvashishtha, I wonder why this PR contains changes in this file, though it shouldn't, because the changes were preliminary merged into master as part of another PR.

Reply:
Yes, I don't understand. Git blame correctly points to #3770 for these lines.